src/main/java/core/basesyntax/RegistrationException.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public class RegistrationException extends RuntimeException {
+    public RegistrationException(String error) {
```

Comment:
Consider renaming the parameter 'error' to 'message' to better represent its purpose and align with common naming conventions.